index.html

```diff
@@ -383,7 +383,7 @@ <h2>Chatbot</h2>
   <ul class="chatbox">
     <li class="chat incoming">
       <span class="material-symbols-outlined">smart_toy</span>
-      <p>Hi there <br>How can I help you today?</p>
+      <p>Hi,<br>How can I help you today?</p>
```

Comment:
@PhodzoW0l7 
i think it will be better if you add "there" hahahahaha, like 
"hi, there"